lib/perl/Genome/Model/ClinSeq/Command/UpdateAnalysis.pm

```diff
@@ -194,6 +194,15 @@ class Genome::Model::ClinSeq::Command::UpdateAnalysis {
               is => 'Boolean',
               doc => 'Allow imported instrument data to be included',
         },
+        validation_as_exome => {
+              is => 'Boolean',
+              doc => 'Treat validation data as exome data. Includes instrument data from validation capture in exome reference alignment model.',
+        },
+        my_trsn => {
+              is => 'Text',
+              doc => 'If using validation-as-exome option, option to specify what instrument data to derive the target-set-region-name and region-of-interest from. valid options: exome or validation', 
+              valid_values => ['exome','validation']
+        },
```

Comment:
I combined the parameters and updated the name. It now requires the user to specify exome or validation when using this option but I think that is reasonable.